main/main.go

```diff
@@ -112,19 +110,14 @@ func initLogging() (*os.File, error) {
 		return nil, errors.Wrapf(err, "Failed to create/open %s", logfile)
 	}
 
-	/* Log UTC Time, Date, Time, (w/ microseconds), line number, and make message prefix come right
-	before the message
-	*/
-	loggerFlags := log.LUTC | log.Ldate | log.Ltime | log.Lmicroseconds | log.Lshortfile | log.Lmsgprefix
+	//Sample: [2020-08-18T20:29:16.079902Z] [1.0.0.0] [main.go:148] [INFO]: Test1
+	infoLogger.logger = log.New(io.MultiWriter(file, os.Stdout), "", 0)
```

Comment:
Another way to avoid having an extra function call (`initLogger`) would be to add the operation as part of the initialization. Something like - 
```go
infoLogger.infoLogger = {
		log.New(io.MultiWriter(file, os.Stdout), "", 0), infoOperation
	}
```